samples/matter/closure/sample.yaml

```diff
@@ -0,0 +1,49 @@
+sample:
+  description: Matter Closure sample
+  name: Matter Closure
+common:
+  tags:
+    - sysbuild
+    - ci_samples_matter
+tests:
+  sample.matter.closure.debug:
+    sysbuild: true
+    build_only: true
+    extra_args:
```

Comment:
This extra_args part can be removed (together with the CONFIG below) for this and .release variants.